srcpkgs/ktorrent/template

```diff
@@ -1,20 +1,27 @@
 # Template file for 'ktorrent'
 pkgname=ktorrent
-version=5.2.0
+version=20.12.1
 revision=1
 build_style=cmake
+configure_args="-DWITH_SYSTEM_GEOIP=ON"
 hostmakedepends="extra-cmake-modules qt5-host-tools qt5-qmake gettext"
 makedepends="qt5-devel qt5-script-devel ki18n-devel kio-devel kxmlgui-devel
  knotifications-devel ksolid-devel kconfig-devel kcoreaddons-devel kdoctools-devel
  kross-devel kcmutils-devel sonnet-devel kdbusaddons-devel kcrash-devel
  karchive-devel knotifyconfig-devel libktorrent-devel kplotting-devel
- gmp-devel taglib-devel kdnssd-devel kdewebkit-devel"
+ gmp-devel taglib-devel kdnssd-devel kdewebkit-devel geoip-devel syndication-devel"
+depends="geoip-data"
 short_desc="BitTorrent client based on the KDE platform"
 maintainer="Norbert Vegh <vegh@norvegh.com>"
 license="GPL-2.0-or-later"
 homepage="https://www.kde.org/applications/internet/ktorrent"
-distfiles="${KDE_SITE}/ktorrent/${version}/ktorrent-${version}.tar.xz"
-checksum=d1031d745c1ace02d11b59c426610aee779d208e3d1ae81f8559c265173732ff
+distfiles="${KDE_SITE}/release-service/${version}/src/ktorrent-${version}.tar.xz"
+checksum=8c5f52d9b4597c117c0ed189c2ada3b9716bc0d5ceb53fde66f6a009bee68354
+
+do_check() {
+	# XX qt.qpa.xcb: could not connect to display
```

Comment:
see `sonnet` template

Reply:
Oh, thank you :) Ι am building it locally with the change!

Reply:
Pushed the change, thanks :)